subsys/fs/pm.yml

```diff
@@ -2,6 +2,6 @@
 
 #ifdef CONFIG_FILE_SYSTEM_LITTLEFS
 littlefs_storage:
-  placement: {after: [mcuboot_storage, app]}
+  placement: {after: [mcuboot_scratch, mcuboot_secondary, app]}
```

Comment:
Have you tried {before: end} ? It could give the same result, and might look prettier.

Reply:
Indeed `{before: [end]}` does the same